dart-github-dart-lang/codereview.settings

```diff
@@ -1,3 +0,0 @@
-CODE_REVIEW_SERVER: http://codereview.chromium.org/
```

Comment:
Should we instead update this to direct to gerrit?

Reply:
I can't think of one stand-alone package that we use gerrit for...

Reply:
I prefer github review myself; no real opinion for the template.